requirements-dev.txt

```diff
@@ -12,7 +12,7 @@ pytest-cov==2.7.1
 requests-mock==1.6.0
 testfixtures==6.8.2
 
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.3.0#egg=digitalmarketplace-test-utils==2.3.0
+git+https://github.com/alphagov/digitalmarketplace-test-utils.git@ris-sf-allow-declaration-reuse#egg=digitalmarketplace-test-utils==2.6.0
 
```

Comment:
`ris-sf-allow-declaration-reuse` ;)

Reply:
Yeah it's to allow people to test it - haven't merged it to test-utils yet.